app/controllers/v0/upload_supporting_evidences_controller.rb

```diff
@@ -4,22 +4,68 @@
 
 module V0
   class UploadSupportingEvidencesController < ApplicationController
-    include FormAttachmentCreate
```

Comment:
Would it not be possible to continue utilizing this concern and conditionally call `save_lighthouse_attachment` on `create` if `FORM_ATTACHMENT_MODEL == SupportingEvidenceAttachment`?

Reply:
the problem is `FormAttachmentCreate` inherits has some complex inheritance that defines what service it faces (EVSS or lighthouse) at the time of class definition, aka application load.  Therefor we had to remove some nominally harmless dependancies that were actually obfuscating downstream coupling ( I think)

Reply:
Yup, Sam's recollection of this is correct, see https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/7e5c4658bb3877539f4f9fcad82cae95f5421bcd